project/placeholders/dependencies.sbt

```diff
@@ -5,3 +5,14 @@ libraryDependencies := Seq(
   "com.typesafe.play" %% "play-slick" % "[play_slick_version]",
   "com.typesafe.play" %% "play-slick-evolutions" % "[play_slick_version]"
 )
+
+dependencyOverrides ++= Seq(
+  "com.typesafe.play" %% "play" % "[play_version]"
+)
+
+// The selected version of the framework may contain packages with known vulnerabilities
+// Exclude the common ones from the base Docker image as they will be added back when compiling the application
+excludeDependencies ++= Seq(
+  ExclusionRule("com.fasterxml.jackson.core", "jackson-databind"),
+  ExclusionRule("com.google.guava", "guava")
```

Comment:
Even the latest stable version of Play has open CVEs from these 2 dependencies ([reference](https://mvnrepository.com/artifact/com.typesafe.play/play_2.13/2.8.20)). 

The primary goal of this project is to pre-download commonly used packages to speed up the build. Since the application dependencies will ultimately be defined by the application itself, we can exclude those to a get cleaner scan for the base image.